packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -27,96 +26,83 @@ export enum TestRunType {
   Method
 }
 
-export class ApexTestRunner {
```

Comment:
didn't need to be a class.  Only 1 method was using the outlineProvider.

that event emitter was even sillier (letting one method call another, I think to get around somthing not being async).